coops-ui/tests/unit/AnnualReport.spec.ts

```diff
@@ -691,8 +690,14 @@ describe('AnnualReport - Part 3 - Submitting', () => {
 
     // stub address data
     vm.addresses = {
-      deliveryAddress: {},
-      mailingAddress: {}
+      registeredOffice: {
+        deliveryAddress: {},
+        mailingAddress: {}
+      },
+      recordsOffice: {
```

Comment:
This doesn't apply to Coops, right? Please remove.

Reply:
Good catch, will remove!

Reply:
Done, thanks.